README.md

```diff
@@ -2,14 +2,17 @@
 
 # Weather App
 
-This is a website for  travellers or anyone who is interested in knowing the weather condition enthusiasts. the data is fetched from accurate live data from API. This web application is for individual, commercial and scientific  uses. The application will allow users to view the current weather conditions. This application is built with JavaScript, HTML, CSS and React JS.more here https://www.loom.com/share/b475c651da16461296bbd861d35e2b68
+This is a website for  travellers or anyone who is interested in knowing the weather condition enthusiasts. the data is fetched from accurate live data from API. This web application is for individual, commercial and scientific  uses. The application will allow users to view the current weather conditions. This application is built with JavaScript, HTML, CSS and React More here https://www.loom.com/share/b475c651da16461296bbd861d35e2b68
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
+    -  [🚀 Live Demo ](#-live-demo-)
+  - [🎥 Video Link ](#-video-link-)
```

Comment:
- [ ] In the Acknowledgments section of the README file, Please make sure to give credit to [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio) for the original design idea as requested in the project requirements.

Reply:
- [OPTIONAL] I highly recommend you mention the name of the designer Nelson Sakwa not only the link to his profile.